maths/monte_carlo_integration_univariate.py

```diff
@@ -0,0 +1,116 @@
+"""
+This program finds the approximate integration value/area under the curve of
+a specified function within specified limits using Monte Carlo integration method.
+
+Further, a graph of the individual areas under the curve considered for the calculation
+is also plotted. (PLOT SECTION -> Optional implementation)
+
+Link to source material:
+https://towardsdatascience.com/the-basics-of-monte-carlo-integration-5fe16b40482d
+
+"""
+
+import doctest
+import random
+
+import numpy as np
+
+
+def function_to_be_integrated(univariate_variable: int) -> float:
+
+    # Doctest
+    """
+    Function to calculate the sin of a particular value of x
+
+    :param univariate_variable: int
+    :return: float
+
+    >>> round(function_to_be_integrated(0))
+    0
+    """
+
+    return np.sin(univariate_variable)
+
+
+def monte_carlo(lower_limit: int, upper_limit: int, number_of_sections: int) -> float:
+
+    # Doctest
+    """
+    Monte Carlo integration function
+
+    :param lower_limit: int
+    :param upper_limit: int
+    :param N: int
+    :return: float
+
+    >>> round(monte_carlo(0, np.pi, 1000))
```

Comment:
Perhaps you could write some more doctests

Reply:
added a few more doctests

Reply:
Please add doctests where lower_limit is negative, a float, a string.  Same for upper_limit and number_of_sections.  Show that the function will raise Exceptionswhen handed garbage input.

Reply:
Required doctests and additional changes have been added